release-notes/next.yml

```diff
@@ -0,0 +1,70 @@
+- release: "35.20220313.1.0"
```

Comment:
(After adding the top-level `releases` key, ) instead of making this an array of objects, WDYT about making it an object instead? E.g.

```yaml
releases:
  35.20220313.1.0:
    issues:
      - id: ...
        title: ...
```

That should make it easier on the website side because it doesn't have to look for the entry matching the release it's rendering.